vtkext/private/module/vtkF3DImguiConsole.cxx

```diff
@@ -0,0 +1,191 @@
+#include "vtkF3DImguiConsole.h"
+
+#include <vtkCommand.h>
+#include <vtkObjectFactory.h>
+
+#include <imgui.h>
+
+#include <array>
+
+struct vtkF3DImguiConsole::Internals
+{
+  enum class LogType
+  {
+    Log,
+    Warning,
+    Error,
+    Typed
```

Comment:
rely on F3DLog enum instead ?

Reply:
I can't because there's a new value `Typed` which does not correspond to a log level but the command typed by the user.